app/assets/javascripts/tabs.js

```diff
@@ -64,19 +66,35 @@
 
       })
 
+     Array.prototype.forEach.call(tabLinks, function (listItem) {
```

Comment:
trivial: if `tabLinks` is an array, I think you can just call the `forEach` on it directly, e.g. `tabLinks.forEach(function(listItem) {...`

Reply:
This actually looks like a duplicate of lines 60-67

Reply:
Did you mean to use `listItem` instead of `tabLinks`?

Reply:
Indentation is a little confusing here, making it look like this is not within the block started on line 54